packages/samples/headless-react/src/App.tsx

```diff
@@ -75,6 +76,7 @@ const {autoUpdateURI: startUpdatingURI} = bindSearchParametersToURI(engine);
 
 function App() {
   useEffect(() => startUpdatingURI(), []);
+  buildContext(engine).add('pageId', 'search page');
```

Comment:
I like how short this is, but if we could have it in a dedicated file, it would be easier to display in the reference.

Since it is so short, maybe we can also show the `.set` method which allows passing an object?

Also, once we have a file, it would be great to specify it in the doc-parser.ts configuration for Context controller.